docs/application/web/api/index.md

```diff
@@ -27,7 +27,6 @@ The following API references provide descriptions of all the APIs that can be us
   <li><a href="3.0/device_api/mobile/index.html" target="_blank">3.0</a></li>
   <li><a href="2.4/device_api/mobile/index.html" target="_blank">2.4</a></li>
   <li><a href="2.3.2/device_api/wearable/index.html" target="_blank">2.3.2</a></li>
-  <li><a href="https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/2.3.1/org.tizen.native.mobile.apireference/index.html" target="api">2.3.1</a>&nbsp;&nbsp;&nbsp;<strong>You can see 2.3.1 on Tizen Developer Site(https://developer.tizen.org)</strong></li>
-  <li><a href="https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/2.3/org.tizen.native.mobile.apireference/index.html" target="api">2.3</a>&nbsp;&nbsp;&nbsp;<strong>You can see 2.3 on Tizen Developer Site(https://developer.tizen.org)</strong></li>
+  <li><a href="2.3.1/device_api/mobile/index.html" target="_blank">2.3.1</a></li>
+  <li><a href="2.3/wearable_api_reference.htm" target="_blank">2.3</a></li>
```

Comment:
I wanted to link mobile API reference, but there is a link error in `mobile_api_reference.htm` file.